README.md

```diff
@@ -546,17 +546,30 @@ which help you to use the various OpenAPI 3 Authentication mechanism.
 
 ## Extensions
 
+| **Extensions**               |    **net/http**    |      **Echo**      |       **Chi**      |       **Gin**      |
```

Comment:
I've decided to include this table to indicate which extensions are supported for each server.

My reasoning is this: It's easier to contribute to one server implementation than several (as no one is using all of them at once and has a know-how required), and a table like this allows to it explicitly

Also, missing server extensions are a "good first issue", because you can use original feature PR as a reference on what needs to be done.